### ISSUE TITLE
Use  complexes directly in `abstract_sequence`

### ISSUE THREAD

**Issue by xtrojak** [sybila/eBCSgen#89]
Allow usage of complexes directly in abstract_sequence - currently only aliases in place of complex are allowed

**Comment by xtrojak** [comment]
Example what is allowed now:

```
#! rules
S{i}:A():A2:cell => ...

#! complexes
A2 = A().A()
```

What is not allow but we want to allow it:

```
#! rules
S{i}:A():A().A():cell => ...
```

The problem should be located somewhere here:
https://github.com/sybila/eBCSgen/blob/734fdfb1bd6a2ffc4fa4e049a400f52b772ee095/eBCSgen/Parsing/ParseBCSL.py#L141
we allow `cmplx_name` and `VAR`, but a complex directly.

**Comment by xtrojak** [comment]
Fixed in #92.
